### PR TITLE
[10.0] [ADD] analytic_plan

### PR DESCRIPTION
**Pull request by nikul-serpentcs**
Analytic Plan
=============

An effective planning of costs and revenues associated with projects or to other analytic accounts becomes essential in organizations that are run by projects, or profit center accounting. The process of cost planning generally follows an rolling wave planning approach, in which the level of detail of the planned costs is increases over time, as the details of the work required are known to the planning group.

### CODE REVIEW

**Comment by nikul-serpentcs** [comment]
@aheficent Could you please review.
Commits are already preserved and can you please check the [history](https://github.com/nikul-serpentcs/pmis/commits/10.0-analytic_plan/analytic_plan).

**Comment by aheficent** [comment]
Can you update the readme to not look ugly? 
image

**Comment by AaronHForgeFlow** [comment]
Can you update the readme to not look ugly? 

image

File: analytic_plan/models/account_analytic_account.py [hunk 1]
```diff
@@ -75,208 +74,14 @@ def _compute_debit_credit_bal_qtty_plan(self):
...
-
     active_analytic_planning_version = fields.Many2one(
         'account.analytic.plan.version',
         'Active planning Version',
         required=True,
         default=default_version
     )
-
-
-class AccountAnalyticLine(models.Model):
```
Comment (AaronHForgeFlow) [inline analytic_plan/models/account_analytic_account.py:92]: The class should be AccountAnalyticLinePlan

**Comment by AaronHForgeFlow** [comment]
The rest looks fine :+1:

**Comment by nikul-serpentcs** [comment]
@aheficent Could you please review again.

**Review by AaronHForgeFlow — APPROVED** [review]
It's ok to me now. Can you squash the commits?

**Comment by nikul-serpentcs** [comment]
@aheficent commits are squashed.

**Pull request merged**